PROJECT

```diff
@@ -14,7 +14,7 @@ resources:
   domain: kubeagi.k8s.com.cn
   group: arcadia
   kind: Laboratory
-  path: github.com/kubeagi/arcadia/api/v1alpha1
+  path: github.com/kubeagi/arcadia/pkg/api/v1alpha1
```

Comment:
@nkwangleiGIT  since we use operator-sdk to manage our apis, it's better we use the operator-sdk's default project sturcture. Perhaps we can manage this operator under a `operator` direcotry. Just like kuberay

Reply:
move api/controllers/main back to follow operator standard, we'll probably move bff graphql and data-processing out of this project later.